### ISSUE TITLE
return await inside an if throw an error

### ISSUE THREAD

**Issue by dmail** [rpetrich/babel-plugin-transform-async-to-promises#43]
Using your babel plugin I had sometime the error 
`TypeError: Cannot read property 'await' of null`.

I figured how to reproduce with the following code:

```js
const { transform } = require("@babel/core")
const transformAsyncToPromises = require("babel-plugin-transform-async-to-promises")

transform(
  `async function test() {
  if (value) {
    return await 12
  }
  return await 10
}`,
  {
    plugins: [transformAsyncToPromises],
  },
)
```

Is this a bug or am I doing something wrong ?

**Comment by dmail** [comment]
For the record you can avoid the error by replacing

```js
return await 12
```

By

```js
const whatever = await 12
return whatever
```

**Comment by rpetrich** [comment]
Appears to be closed by #41
